presto-main/src/main/java/com/facebook/presto/server/PluginManager.java

```diff
@@ -65,7 +65,7 @@
 {
     private static final ImmutableList<String> SPI_PACKAGES = ImmutableList.<String>builder()
             .add("com.facebook.presto.spi.")
-            .add("com.fasterxml.jackson.annotation.")
+            .add("com.fasterxml.jackson.")
```

Comment:
Instead of opening up the whole jackson package, we should just give access to the afterburner package.

Reply:
Let's add a comment here saying why `AfterburnerModule.setUseValueClassLoader(false)` doesn't work, and why we need this change, so that the future readers of this code don't get confused.